site/content/events/2016-oslo/index.html

```diff
@@ -3,38 +3,48 @@
 filter:
 - erb
 - markdown
-title: welcome
 layout: event
 dirty: true
 ---
 <% @eventhome = @page.directory.split(File::SEPARATOR)[0..1].join(File::SEPARATOR) %>
 <center>
-  <table><tr><td>
-        <center>
-        <!-- <img border=0 width="160" height="240" src="./yourlogohere.png">
--->
-</td><td valign=top>
-<strong>DevOpsDays is coming to <%= render(:partial => "/#{@eventhome}/_target_location") %>! </strong>
-<br><br>
-Dates: <%= render(:partial => "/#{@eventhome}/_event_date_start") %> - <%= render(:partial => "/#{@eventhome}/_event_date_end") %>
-<br><br>
-<!-- <a href="./location">Location</a> -->
-<br><br>
 
-<!-- <a href="./propose">Propose a talk</a>! -->
-<br><br>
+<table>
+<tr><td><center><img border=0 width="450" height="106" src="./logos/oslo2.png"></td></tr>
 
-<!-- <a href="./registration">Register</a>! -->
-<br><br>
+<tr><td valign=top>
+
+<br>
+
+<h2>DevOpsDays is coming to <%= render(:partial => "/#{@eventhome}/_target_location") %>!</h2>
+
+<p style="font-size:1.3em; font-weight: bold;">
+From <%= render(:partial => "/#{@eventhome}/_event_date_start") %>
+to <%= render(:partial => "/#{@eventhome}/_event_date_end") %>
+</p>
+
+<p>For the first time, DevOpsDays is coming to Oslo!</p>
```

Comment:
Should we add a little explanation of what the conference is about since this is the first time DevOpsDays is held in Oslo? E.g. that we have a mix of soft and hard talks and about everything DevOps and Continuous Delivery.